<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Baker Data Validator</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0" colspan="2">
     <widget class="QCheckBox" name="skip_geometry_errors_check_box">
      <property name="toolTip">
       <string>Ignores geometry errors (--skipGeometryErrors) and AREA topology validation (--disableAreaValidation)</string>
      </property>
      <property name="text">
       <string>Skip Geometry Errors</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QToolButton" name="config_file_tool_button">
      <property name="text">
       <string>...</string>
      </property>
      <property name="popupMode">
       <enum>QToolButton::MenuButtonPopup</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="config_file_line_edit">
      <property name="toolTip">
       <string>ili2db option --validConfig</string>
      </property>
      <property name="placeholderText">
       <string>Validator config file (--validConfig)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QWidget" name="validation_layout" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="run_button">
           <property name="text">
            <string>âœ”</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progress_bar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="result_table_view"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="error_count_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="flash_button">
        <property name="toolTip">
         <string>Highlight current feature on map</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="auto_pan_button">
        <property name="toolTip">
         <string>Automatically pan to the current coordinates / feature</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="auto_zoom_button">
        <property name="toolTip">
         <string>Automatically zoom to the current feature / coordinates</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="info_label">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="export_models_layout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="filter_layout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="activate_verbose_mode_check_box">
      <property name="text">
       <string>Activate Verbose Mode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
